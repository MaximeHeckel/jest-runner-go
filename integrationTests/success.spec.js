const run = require('./run');

it('Works when it has passing tests', () =>
  expect(run('success')).resolves.toMatchSnapshot());
