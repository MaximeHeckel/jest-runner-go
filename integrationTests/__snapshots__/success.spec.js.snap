// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has passing tests 1`] = `
"PASS integrationTests/fixtures/success/main_test.go
  âœ“ /go/src/project/go-test-runner/integrationTests/fixtures/success/main_test.go
Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Ran all test suites."
`;
